dexc/dexc.go

```diff
@@ -0,0 +1,190 @@
+package dexc
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"decred.org/dcrdex/client/asset"
+	"github.com/planetdecred/dcrlibwallet"
+	"github.com/planetdecred/dcrlibwallet/dexdcr"
+
+	"decred.org/dcrdex/client/core"
+	"decred.org/dcrdex/dex"
+)
+
+// Dexc represents of the Decred exchange client.
```

Comment:
This what you mean maybe?
```suggestion
// Dexc represents the Decred Dex client.
```